make/buf/all.mk

```diff
@@ -135,13 +135,6 @@ bufgeneratesteps:: \
 bufrelease: $(MINISIGN)
 	DOCKER_IMAGE=golang:1.21-bullseye bash make/buf/scripts/release.bash
 
-# We have to manually set the Homebrew version on the Homebrew badge as there
-# is no badge on shields.io for Homebrew packages outside of homebrew-core
-
```

Comment:
We don't manually set the badge version anymore. It's now dynamic, see https://github.com/bufbuild/buf/blob/6befbe40183d85d55677092107ff1f1f1ad56a4e/README.md?plain=1#L9